mcs/class/corlib/ReferenceSources/String.cs

```diff
@@ -41,6 +41,11 @@ namespace System
 {
 	partial class String
 	{
+		public String (ReadOnlySpan<char> value)
+		{
+			throw new NotImplementedException ("string ctor");
```

Comment:
what's our plan for implementing the new Span overloads in code we don't share with corefx? can we grab the implementation from coreclr for this?

Reply:
The plan is to switch to CoreFX code where possible. However, in this specific case we need runtime support

Reply:
can we just copy the string out of the span until we have runtime support?

Reply:
that's what this method will do, except it's a constructor on a string type so it's not just copy and paste